### ISSUE TITLE
MoCo tutorial accuracy fails with pl

### ISSUE THREAD

**Issue by philippmwirth** [lightly-ai/lightly#375]
# MoCo tutorial accuracy fails with pl

Pytorch-lightning expects probabilities for the accuracy calculation, see

https://github.com/PyTorchLightning/pytorch-lightning/blob/cf5dc04db4a84e31d8c360c350b3896b67b38aeb/pytorch_lightning/metrics/classification/helpers.py#L55

Our MoCo tutorial outputs logits and hence fails during the validation step.
